metaserver/src/main/scala/scala/meta/languageserver/MSchedulers.scala

```diff
@@ -0,0 +1,23 @@
+package scala.meta.languageserver
+
+import java.util.concurrent.Executors
+import monix.execution.Scheduler
+import monix.execution.schedulers.SchedulerService
+
+/**
+ * Utility to manage monix schedulers.
+ *
+ * @param global The default scheduler when you are unsure which one to use.
+ * @param lsp to communicate with LSP editor client.
+ * @param sbt to communicate with sbt server.
+ */
+case class MSchedulers(global: Scheduler, lsp: Scheduler, sbt: Scheduler)
+object MSchedulers {
+  def apply(): MSchedulers = new MSchedulers(main, lsp, sbt)
+  lazy val main: SchedulerService =
+    Scheduler(Executors.newFixedThreadPool(4))
+  lazy val lsp: SchedulerService =
+    Scheduler(Executors.newFixedThreadPool(1))
+  lazy val sbt: SchedulerService =
```

Comment:
Rename `lsp` to `client` and `sbt` to `server`?

Reply:
My OCD liked having three characters acronyms based on the protocol that each scheduler handles :) "client" and "server" are not so self-documenting because when connecting to bsp/sbt as a "client" we still need to spin up a "server".